configuration/multimedia

```diff
@@ -0,0 +1,127 @@
+---
+layout: doc
+title: How to play multimedia content
+permalink: /doc/config/multimedia/
+---
+
+This short howto will explain how you can use Spotify, Netflix or Amazon Prime under Qubes OS.
+This steps have been tested by providing an own "multimedia" App VM which is based on the Qubes OS Debian 8 Template VM.
+
+Listen to Spotify
+=================
+Install-Howto: https://www.spotify.com/de/download/linux/
+
+Detailed steps:
+
+1) sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys BBEBDCB318AD50EC6865090613B00F1FD2C19886
+2) echo deb http://repository.spotify.com stable non-free | sudo tee /etc/apt/sources.list.d/spotify.list
+3) sudo apt-get update
+4) sudo apt-get install spotify-client
+
+I could then start the native spotify client .. easy.
+
+
+Play Netflix
+============
+You can play Netflix by installing a more recent version of firefox or by installing Google Chroome.
+As Google Chrome will play video from Netflx and (!) Amazon, the suggestion is to install Google Chrome.
+If you don't use Amazon Prime, Firefox will work.
+
+Install Firefox
```

Comment:
Firefox could be installed from a package repository.
Link: https://mozilla.debian.net

To Install Firefox ESR v53.x:
1) Add new repository to the sources:
deb http://mozilla.debian.net/ jessie-backports firefox-esr

2) update repository and install Firefox:
$ apt-get update
$ apt-get install -t jessie-backports firefox-esr

I'll try those steps and if this will work, I'll update the documentation including the correct steps.